### DISCUSSION TITLE
Injecting Logs into Trace

### DISCUSSION THREAD

**Discussion by omerace** [open-telemetry/opentelemetry-dotnet#4118] [General]
Does OpenTelemetry support auto-instrumentation to inject log context to trace data. Via Opentelemetry.logging you can add traceId/spanId information to the log context. However, while tracing the requests, it will be more comprehensible to get log context in jaeger/zipkin UI. If such a feature already exists, I would appreciate it if you could add the links.

**Comment by cijothomas** [comment]
Can you elaborate what is "log context"? Are you referrring to iLogger scopes?
